.vscode/settings.json

```diff
@@ -28,6 +28,9 @@
         "typescriptreact",
     ],
     "eslint.run": "onSave",
+    "editor.rulers": [
```

Comment:
What does these changes do?

Reply:
I find having a ruler for 100 very useful in vscode. It doesn't enforce anything, but when doing code reviews on the github user interface, 100 characters is where it starts wrapping lines. So anything below 100 lines is very neat when doing code reviews.

I try to keep my lines to 100 for this reason - and really want a ruler. I can move it to my personal settings if it's an issue.

Reply:
I'm fine with it, just wasn't expecting it. There's just a trailing comma on the `"editor.formatOnSave": true,` setting below.

Reply:
I love trailing commas. It means when you add another line, the unchanged line doesn't come up as part of the code review.

Reply:
I know..... it's just a visual bug for me so ignore me :)